Library/Homebrew/utils/github.rb

```diff
@@ -140,6 +140,7 @@ def open(url, data=nil)
     begin
       if data
         data_tmpfile.write data
+        data_tmpfile.close
```

Comment:
Could it create problems that the same file is closed a second time further down? Would using `IO#flush` instead of `IO#close` also fix the problem?